### DISCUSSION TITLE
page.twilio.design homepage is not responsive for smaller screens

### DISCUSSION THREAD

**Discussion by josamit** [twilio-labs/paste#3810] [General] [answered]
Hey folks,

I just noticed that responsive behaviour of page.twilio.design is broken on smaller screens. I am using a 14" Macbook pro and the homepage looks like the following. I thought ya'll should know in case this isn't being tracked. 

<img width="1512" alt="image" src="https://github.com/twilio-labs/paste/assets/62208722/3f1516d4-0b86-4c6a-8694-17be0a89909d">

**Comment by serifluous** [comment]
Hey @josamit ! Which part are you referring to as broken?

**Reply by josamit** [reply]
'broken' might be incorrect word here, the layout seems off. Please see the attached image for reference. The main image ya'll are using in the homepage is out of bounds, so when you open paste.twilio.design on smaller screens, half of the page content (from right side) isn't visible at all. 

![image](https://github.com/twilio-labs/paste/assets/62208722/7e67b1bf-a9be-4ceb-9b54-135a0e536157)

**Reply by serifluous** [reply, marked as answer]
ahh got it, yeah that's actually intentional. We don't want it to look broken, though, so good shout—we'll take that into consideration in our next iteration of it. Thanks!
